### DISCUSSION TITLE
Visualization for Segmentation Examples

### DISCUSSION THREAD

**Discussion by paulwingert865** [pyg-team/pytorch_geometric#2575] [Ideas]
Hey there,

I'm trying the Dgcnn and pointnet2 segmentation examples and was wondering what's the best possibilty to visualize the results.

**Comment by rusty1s** [comment]
I think the easiest way to visualize is via `matplotlib.scatter`, see: https://chris35wills.github.io/courses/PythonPackages_matplotlib/matplotlib_3d/
